src/frequenz/channels/util/_merge.py

```diff
@@ -20,6 +20,12 @@ class Merge(Receiver[T]):
         stream, by using `Merge` like this:
 
         ```python
+        from frequenz.channels import Broadcast
+
+        channels = [Broadcast[int]("input-chan")]*2
+        receiver1 = channels[0].new_receiver()
+        receiver2 = channels[1].new_receiver()
```

Comment:
This is not great as an example, because it will end up with 2 channels with the same name, which will be very confusing for debugging. I would expand it:

```suggestion
        chan1 = Broadcast[int]("input-chan-1")
        chan2 = Broadcast[int]("input-chan-2")
        receiver1 = chan1.new_receiver()
        receiver2 = chan2.new_receiver()
```